app/src/androidTest/java/com/appunite/loudius/util/IntegrationTestRule.kt

```diff
@@ -26,10 +27,13 @@ import org.junit.rules.TestRule
 import org.junit.runner.Description
 import org.junit.runners.model.Statement
 
-class IntegrationTestRule(testClass: Any) : TestRule {
+class IntegrationTestRule(
+    testClass: Any,
+    testActivity: Class<out ComponentActivity> = TestActivity::class.java,
```

Comment:
Question: Do we need this because of the deeplink and opening it in MainActivity?

Reply:
yes, it's been changed because we want to use the navigation defined in MainActivity.